tox.ini

```diff
@@ -11,30 +11,35 @@ deps =
     portend
     pytest-services
     munch
+    {[testenv:typing]deps}
+    {[testenv:lint]deps}
 commands =
-    coverage run --source twine -m pytest {posargs:tests}
+    coverage run --source twine -m pytest tests
```

Comment:
Removed the `{posargs}` because it doesn't make sense when running multiple tools.

Reply:
Oh, that's a shame. Support for `{posargs}` is way more valuable than having a single test environment. Removing `{posargs}` means it's not readily possible to `tox -- -k one_test` or `tox -- --lf` (two commands I frequently run to downselect tests).